coops-ui/src/schemas/address-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { required, maxLength } from 'vuelidate/lib/validators'
+// The Address schema containing Vuelidate rules.
+// NB: This should match the subject JSON schema.
+export const officeAddressSchema = {
```

Comment:
Hah. Thanks for changing the object name. Now do you think we should change the filename to `address-schemas.ts` or `office-address-schema.ts`? :)

Reply:
haha your 100% right, i will update that!

Reply:
Done. Thanks.